packages/core/src/amazonqGumby/chat/controller/messenger/messenger.ts

```diff
@@ -376,19 +376,20 @@ export class Messenger {
     ) {
         const buttons: ChatItemButton[] = []
 
+        // don't show these buttons when server build fails
         if (!disableJobActions) {
-            // Note: buttons can only be clicked once.
-            // To get around this, we remove the card after it's clicked and then resubmit the message.
```

Comment:
Not true. We can set `disabled: false` to allow buttons to be re-clicked, removing the need to call `sendJobSubmittedMessage` redundantly on line 386 in the file above.